src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -740,6 +840,68 @@ public TopLevelItem newInstance(ItemGroup parent, String name) {
 
     }
 
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    public static class SampleComputedFolderWithWorkflowJobAsChildren extends ComputedFolder<WorkflowJob> {
```

Comment:
What is the point of the `workflow-multibranch` test dep if you are not actually creating a `MultiBranchProject`?

Reply:
None indeed. Dependencies cleaned.